bitmark-references/bitmark-cli/bitmark-cli.md

```diff
@@ -0,0 +1,703 @@
+                                 that block will be sent to the new
+                                 address.
+
+`--litecoin|-l ADDRESS`          A valid Litecoin address for the corresponding
+                                 network. All future earnings from
+                                 that block will be sent to the new
+                                 address.
+-------------------------------  ---------------------------------
+
+### EXAMPLE
+
+~~~
+bitmark-cli --identity=fred --network=testing blocktransfer --txid=
+~~~
+
+
+## provenance
+
+Display the provenance starting from a particular transaction id back
+in time towards the issue record.  Since the record count is limited
+it may be necessary to use the last transaction id with a new call to
+`provenance` to display more records; then repeat until the asset
+record is output.
+
+### OPTIONS
+
+------------------------------  ---------------------------------
+`--txid|-t HEX-ID`              The transaction id from a previous issue or
+                                transfer.
+
+`--count|-c NUMBER`             Limit the number of provenance records displayed.
+                                This defaults to 20 records if omitted
+                                and can be set from 1 to 100.
+------------------------------  ---------------------------------
+
+### EXAMPLE
+
+~~~
+bitmark-cli --identity=fred --network=testing provenance --txid=8981eb58c965e2360b3ffeedf47d8f770a2fff7f67e5c348302d839afbf83dee --count=10
+~~~
+
+
+## owned
+
+Display the ownership records for a given identity or account
+
+### OPTIONS
+
+------------------------------  ---------------------------------
+`--owner|-o IDENTITY|ACCOUNT`   To list records for a different account
+                                either by an identity or a Base58
+                                account string.  The default is to use
+                                the global identity.
+
+`--start|-s NUMBER`             Specify another start point rather the the default
+                                of zero.  The output JSON includes a
+                                field `next` which if non-zero means
+                                there are more records to display.
+                                Just use the value of `next` in
+                                subsequent `owned` commands to page
+                                though all ownership records.
+
+`--count|-c NUMBER`             Limit the number of ownership records displayed.
+                                This defaults to 20 records if omitted
+                                and can be set from 1 to 100.
+------------------------------  ---------------------------------
+
+### EXAMPLE
+
+~~~
+bitmark-cli --identity=fred --network=testing owned
+bitmark-cli --identity=fred --network=testing owned --owner barney
+bitmark-cli --identity=fred --network=testing owned --owner eywWbSGcYaYPkY4e2aerGWRs1fNoaSDxh3BgLVENgBZmhTsdLz
+~~~
+
```

Comment:
This is another example of where it'd be good to show output, because with output, you could show a list of OWNED that's truncated by the COUNT, then show using the start option to list the NEXT records.